v2/decoder_operator.go

```diff
@@ -0,0 +1,59 @@
+package jwt
```

Comment:
copyright banner.. will not repeat, please check all files under `v2/` directory.